test/WebJobs.Script.Tests/Configuration/FunctionsHostingConfigurationTest.cs

```diff
@@ -43,30 +41,24 @@ public void FunctionsHostingConfiguration_Registered_ByDefault()
         }
 
         [Fact]
-        public void FunctionsWorkerDynamicConcurrencyEnabled_Throws_InvalidOperationException()
-        {
-            FunctionsHostingConfiguration conf = new FunctionsHostingConfiguration(_environment, _loggerFactory, Path.Combine("C:\\", "test.txt"), DateTime.Now.AddMilliseconds(1), TimeSpan.FromMinutes(5));
-            Assert.Throws<InvalidOperationException>(() => conf.FunctionsWorkerDynamicConcurrencyEnabled);
-        }
-
-        [Fact]
-        public async Task FunctionsWorkerDynamicConcurrencyEnabled_UpdatesSettings()
+        public async Task GetValue_UpdatesSettings()
```

Comment:
nit: GetValue doesn't update any settings - it just returns a value, right. Maybe the test should be named GetValue_ReturnsExpectedValue?

Reply:
The idea of the test is checking that the settings are updated if the config file was rewritten. 
In the test we write to the config file twice and set short cache time for the settings.